apps/docs/docs/dev/04-guides/06-jayvee-extensions.md

```diff
@@ -13,55 +13,44 @@ Jayvee extensions that shall be used by default are bundled into the so-called [
 
 ### Language extension
 
+:::info
+This mechanism is currently under construction! We are moving from a code-first approach (using typescript classes to generate the Jayvee model of the blocktype) to a model-first approach (parsing the Jayvee model of the blocktype and creating ast wrappers). 
+:::
```

Comment:
We should not use these info boxes!

Reply:
Addressed in [48b883c](https://github.com/jvalue/jayvee/pull/445/commits/48b883c15d6416e5e6006d7e6cff186f6decb8d9)